packages/eas-cli/src/build/runBuildAndSubmit.ts

```diff
@@ -106,7 +106,10 @@ export async function runBuildAndSubmitAsync(
   projectDir: string,
   flags: BuildFlags,
   actor: Actor,
-  getDynamicPrivateProjectConfigAsync: DynamicConfigContextFn
+  getDynamicPrivateProjectConfigAsync: DynamicConfigContextFn,
+  buildProfilesOverride?: ProfileData<'build'>[],
+  downloadSimBuildAutoConfirm?: boolean,
```

Comment:
nit: prefer to refactor the params here to be destructured object (can use vs code refactor thing) now that we're adding a boolean param. This way we get a named boolean param at the callsite.